dist/toolwindow.js

```diff
@@ -515,7 +509,6 @@ function _defineProperty(obj, key, value) { if (key in obj) { Object.definePrope
       moveToConfiguredStartPosition: function moveToConfiguredStartPosition() {
         if (this._isAutoPosition) {
           return;
-          gg;
```

Comment:
I've no idea why this line was in the dist-code - it does not exist in the lib-code.

Reply:
This is probably my fault, whilst attempting to merge back from dist to lib -- `gg` in vim would take you to the top of the file, if in normal mode. `dist` should be considered an artifact of the truth in `lib`. Normally I wouldn't even check it in, but I figured it might be useful for someone to be able to link to the built .js